### ISSUE TITLE
No found /img_features/depth_14x14.hdf5

### ISSUE THREAD

**Issue by weimingz996** [MarSaKi/VLN-BEVBert#10]
![image](https://github.com/MarSaKi/VLN-BEVBert/assets/136681140/26744d85-1dcf-47f9-a9f0-3b8c1229e8e8)
Hi，
Can you upload this file？thanks!
By the way, can you show me your all file hierarchical structures for me（It can be screenshots）？

**Comment by yueluhhxx** [comment]
Thank you for sharing such an interesting job. When I tried to execute the 'CUDA_VISIBLE_DEVICES=0,1,2,3 bash run_pt/run_r2r.bash 2333' command, I found that I was also missing the file named 'depth_14x14.hdf5'. Could you please tell me where I can get this file? Look forward to your reply~

**Comment by yueluhhxx** [comment]
> Thank you for sharing such an interesting job. When I tried to execute the 'CUDA_VISIBLE_DEVICES=0,1,2,3 bash run_pt/run_r2r.bash 2333' command, I found that I was also missing the file named 'depth_14x14.hdf5'. Could you please tell me where I can get this file? Look forward to your reply~

I guess this is because I didn't execute the "python precompute_features/grid_depth.py" command before performing the training on R2R-CE.

**Comment by MarSaKi** [comment]
Yes, you are right, just run  precompute_features/grid_depth.py to get this file.
Thx,
